app/controllers/admin/add_kudos_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admin
+  class AddKudosController < AdminController
```

Comment:
I would consider moving it to the `EmployeesController` - the reason is that we are not working with the `AddKudos` model, we are updating attributes of `Employees` - personally I would use custom action here.

Reply:
very good advice, nicely explained :D